<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stupage</class>
 <widget class="QWidget" name="stupage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="boxSch">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入书名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnsch">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>540</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QTableView" name="bookTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>800</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bnoBox">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>190</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>194</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>书序列号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="borrowbtn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>330</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>借书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showhisBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>显示记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnbtn">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>330</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>还书</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
